samples/subsys/ipc/openamp/remote/src/main.c

```diff
@@ -73,9 +73,17 @@ static u32_t virtio_get_features(struct virtio_device *vdev)
 
 static void virtio_notify(struct virtqueue *vq)
 {
+#if defined(CONFIG_SOC_MPS2_AN521) || \
+	defined(CONFIG_SOC_V2M_MUSCA_A) || \
+	defined(CONFIG_SOC_V2M_MUSCA_B1)
+	u32_t current_core = sse_200_platform_get_cpu_id();
+
+	ipm_send(ipm_handle, 0, current_core ? 0 : 1, 0, 1);
```

Comment:
Why does the data here mater?

Reply:
CPU0 send notification to CPU1, vise vesa.

Reply:
The dummy data is not necessary here for MHU based hardware. But it needs to distinguish the current core No. for notifications.